### PR TITLE
site(about): add to menu, add opencollective and stats badges

### PR DESCRIPTION
**Pull request by curbengh**
## Check List

- [x] Others (Update, fix, translation, etc...)

### CODE REVIEW

**Comment by SukkaW** [comment]
Is it possilble to load star & download stat from `img.shields.io` so we don't have to update them frequently?

**Comment by SukkaW** [comment]
39k downloads per month

28k stars

**Comment by curbengh** [comment]
> `https://img.shields.io/npm/dm/hexo?style=flat-square&label=npm%20downloads`
> `https://img.shields.io/github/stars/hexojs/hexo?style=social`

Just added

File: source/about/index.md [hunk 1]
```diff
@@ -2,32 +2,33 @@
...
-{% note info Bitcoin %}
-<p align="center">
-![QR Code](bitcoin_qr.png)
-[3M9wCDjRCSkj2ecwuZc3ReJETWRow43UYg](bitcoin:3M9wCDjRCSkj2ecwuZc3ReJETWRow43UYg)
-</p>
-{% endnote %}
+[![Backers](https://opencollective.com/hexo/tiers/backers.svg?avatarHeight=60&width=600)](https://opencollective.com/hexo)
+
+[![QR Code](bitcoin_qr.png)](bitcoin:3M9wCDjRCSkj2ecwuZc3ReJETWRow43UYg)
+BTC: 3M9wCDjRCSkj2ecwuZc3ReJETWRow43UYg
```
Comment (SukkaW) [inline source/about/index.md]: BTC address could be wrapped in code blocks:

```suggestion
BTC: `3M9wCDjRCSkj2ecwuZc3ReJETWRow43UYg`
```

**Comment by curbengh** [comment]
Updated some sentence. I removed new lines from badges, so that they display side-by-side (hopefully).

**Comment by NoahDragon** [comment]
lol, I never expected we will publish the about page. As we have OpenCollective to accept donations, I would suggest we can remove the Bitcoin link.

**Comment by curbengh** [comment]
The about page has been published many [moons ago](https://github.com/hexojs/site/commit/80c11385286d50e152ebd5c835000a3bd7279c3a#diff-5bf2295dfb6f7a330d12be5a78b4606f), just never linked on the menu. any project website should have about page anyway.

I removed the btc in https://github.com/hexojs/site/pull/1145.

**Pull request merged**